app/webpacker/components/RegistrationsV2/Registrations/Competitors.jsx

```diff
@@ -1,26 +1,95 @@
 import { useQuery } from '@tanstack/react-query';
-import React, {
-  useMemo,
-  useReducer,
-} from 'react';
+import React from 'react';
 import {
   Flag, Segment, Table,
 } from 'semantic-ui-react';
 import _ from 'lodash';
+import {
+  flexRender, getCoreRowModel, getSortedRowModel, useReactTable
+} from '@tanstack/react-table';
 import {
   getConfirmedRegistrations,
 } from '../api/registration/get/get_registrations';
-import createSortReducer from '../reducers/sortReducer';
 import EventIcon from '../../wca/EventIcon';
 import { personUrl } from '../../../lib/requests/routes.js.erb';
 import I18n from '../../../lib/i18n';
 import { countries } from '../../../lib/wca-data.js.erb';
-import { getPeopleCounts, getTotals, getUserPositionInfo } from './utils';
+import { expandSortTerm, getPeopleCounts, getTotals, getUserPositionInfo } from './utils';
 import PreTableInfo from './PreTableInfo';
 import Errored from '../../Requests/Errored';
 import Loading from '../../Requests/Loading';
 
-const sortReducer = createSortReducer(['name', 'country', 'total']);
+const getColumns = (eventIds) => [
+  {
+    accessorKey: 'user.name',
+    header: I18n.t('activerecord.attributes.registration.name'),
+    cell: (info) => info.getValue() ? (
+      <a href={personUrl(info.row.original.user.wca_id)}>
+        {info.getValue()}
+      </a>
+    ) : (
+      info.getValue()
+    ),
```

Comment:
I don't get this with regards to the ternary condition. If `getValue` is truthy, you return an `<a>` link, but if it's falsey you just return... that falsey value explicitly? Why not just `info.getValue() && (<a>)`?

Reply:
Pretty sure I just put the wrong condition, it should be checking that the wca id exists.